app/messages/Messages.scala

```diff
@@ -1318,9 +1323,9 @@ object Messages {
       welsh   = "Yr hyn sy’n digwydd nesaf"
     )
 
-    val `HMRC will send you a letter within 5 working days with your payment dates.`: Message = Message(
-      english = "HMRC will send you a letter within 5 working days with your payment dates.",
-      welsh   = "Bydd CThEF yn anfon llythyr atoch cyn pen 5 diwrnod gyda’ch dyddiadau talu."
+    val `We will send a letter by post confirming the set up of your Direct Debit instruction within 5 working days.`: Message = Message(
```

Comment:
Same with this

Reply:
Confirmation page prototype:
https://supp-prototype.herokuapp.com/3-1/confirmation

Karen just confirming with business if wording needs to be applied for both EPAYE & VAT. For now, updating. If need to make them separate, can do that later.